### DISCUSSION TITLE
Not able to build iOS app for different flavors

### DISCUSSION THREAD

**Discussion by UsamaLaeeqTech** [codemagic-ci-cd/codemagic-docs#2308] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Create your development workflow.
2. Configure all the settings.
3. Start new build.

### Expected results

Android apk  and iOS app are  being successfully created.

### Actual results

4. Android apk is being successfully created but iOS one is not. Getting this error
"Runner" requires a provisioning profile with the Push Notifications and Sign in with Apple features. Select a provisioning profile in the Signing & Capabilities editor.
/Users/builder/clone/ios/Runner.xcodeproj.
whereas all the configurations in Xcode are okay.

**Comment by himesh-cm** [comment]
Hi @UsamaLaeeqTech,
As the error suggests, can you confirm if you enabled `push notifications` in your Provisioning profile?
You should see it below in your Apple developer account under the `Enabled Capabilities` section.

<img width="479" alt="Screenshot 2024-01-15 at 17 30 30" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/106069365/b10d8752-8abd-407d-b062-31e8d47e0e58">

**Reply by UsamaLaeeqTech** [reply, marked as answer]
Hi @himesh-cm yep I've added those , I just passed the flavor in build arguments and it worked
